src/components/events/create-event/advanced-options/AddIngredient.js

```diff
@@ -0,0 +1,175 @@
+  });
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setFormInput({ ...formInput, [e.target.name]: e.target.value });
+  };
+
+  const submitIngredient = (e) => {
+    e.preventDefault();
+    const newIngredient = {
+      id: ingredientList.length + 1,
+      name: formInput.name,
+      quantity: formInput.quantity,
+      measurement: formInput.measurement,
+    };
+    setIngredientList([...ingredientList, newIngredient]);
+    setFormInput({ name: "", quantity: "", measurement: "" });
+  };
+
+  const removeIngredient = (id) => {
+    setIngredientList(ingredientList.filter((item) => item.id !== id));
+  };
+
+  return (
+    <div style={{ marginTop: "55px", marginBottom: "25px" }}>
+      <Typography style={{ marginBottom: "10px" }}>
+        Request for guests to bring ingredients
+      </Typography>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          border: "2px solid #f0f0f0",
+          borderRadius: "15px",
+        }}
+      >
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            width: "100%",
+          }}
+        >
+          <div
+            style={{
+              display: "flex",
+              width: "100%",
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <div className="createFormInputDiv" style={{ width: "30%" }}>
+              <input
+                type="text"
+                name="name"
+                value={formInput.name}
+                onChange={handleChange}
+                placeholder="Ingredient"
+              />
+            </div>
+
+            <div
+              className="createFormInputDiv"
+              style={{ width: "10%", marginLeft: "2%" }}
+            >
+              <input
+                type="numbers"
+                name="quantity"
+                value={formInput.quantity}
+                onChange={handleChange}
+                min="0"
+                step="0.01"
+                placeholder="Quantity"
+              />
+            </div>
+
+            <div className="eventTimeDiv" style={{ marginLeft: "2%" }}>
+              <label>
+                Measurement
+                <Select
+                  name="measurement"
+                  value={formInput.measurement}
+                  onChange={handleChange}
+                  id="greenSelect"
+                  disableUnderline={true}
+                >
+                  <MenuItem value=""></MenuItem>
+                  <MenuItem value={"cup"}>cup</MenuItem>
+                  <MenuItem value={"cups"}>cups</MenuItem>
+                  <MenuItem value={"gallon"}>gallon</MenuItem>
+                  <MenuItem value={"gallons"}>gallons</MenuItem>
+                  <MenuItem value={"lbs"}>lbs</MenuItem>
```

Comment:
This is the only item that is abbreviated... Please spell it out as Pounds and also make another entry as Pound.